### ISSUE TITLE
a11y: Color Contrast - Secret Create

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13808]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1
Main Content
- More info link
- Cancel button

Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 4.48:1
Main Content
- Cancel button (On focus)"

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 171 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #2F68DF
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> - Cancel button (on focus and hover)
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.
> "
